src/CampaignAdmin.php

```diff
@@ -448,7 +448,8 @@ public function readCampaign(HTTPRequest $request)
             }
 
             /** @var ChangeSet $changeSet */
-            $changeSet = ChangeSet::get()->byID($request->param('ID'));
```

Comment:
Can you add a unit test to cover this scenario

Reply:
DONE